apps/src/templates/teacherDashboard/SectionRow.jsx

```diff
@@ -136,22 +148,26 @@ class SectionRow extends Component {
   onClickDeleteNo = () => this.setState({deleting: false});
 
   onClickDeleteYes = () => {
-    const { section, removeSection } = this.props;
+    const { sections, sectionId, removeSection } = this.props;
+    const section = sections[sectionId];
     $.ajax({
       url: `/v2/sections/${section.id}`,
       method: 'DELETE',
     }).done(() => {
       removeSection(section.id);
     }).fail((jqXhr, status) => {
-      // TODO(bjvanminnen): figure out how what we want to do in this case
+      // We may want to handle this more cleanly in the future, but for now this
+      // matches the experience we got in angular
+      alert(i18n.unexpectedError());
```

Comment:
Should we set `deleting` back to false in this situation, so we ask for confirmation again?